go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -493,9 +503,11 @@ func addDefaultConntrackRules(nodeName, gwBridge, gwIntf string, stopChan chan s
 	}
 	nFlows++
 
-	// table 1, all other connections do normal processing
+	// table 1, unknown L3/L4 traffic go to host
+	// we have already CT'ed this IP traffic, so we know it did not belong to OVN backed k8s service
+	// and we have no match so we know it must not be for OVN
```

Comment:
Comment is confusing... I think you're saying "The traffic isn't established or related, and didn't match any rule created by addService() so we know it must not be for OVN" ?

Hm... but assuming we send this packet to the host, and it results in a connection being established, then presumably the next packet in the same connection is going to come onto the bridge via the same path, but that time it will match the `ct_state=+trk+est` rule above and get output to `ofportPatch`, right?

Reply:
I'll clear up the comment:
"This must be IP/IPv6 traffic that is not return traffic to OVN"

To your 2nd question, we dont CT commit packets coming into br-ex from the physical port or from the host. So there is no ct sessions for that traffic. There's only sessions stored for OVN -> external.

Reply:
> we dont CT commit packets coming into br-ex from the physical port or from the host.

ah, the comment says "CT" not "CT commit"... ok, so the packet we are processing here will have been sent to conntrack by the rule in table 0, but nothing ever _commits_ that conntrack state, so when the next packet comes, it will also end up at this rule. got it. (but please improve the comment)